### PR TITLE
Address PED issues

### PR DESCRIPTION
**Pull request by sibinhho99-nus**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -276,9 +276,10 @@ Adds a patient's medical information to MedBook.
 Format: `add t/medical i/NRIC [a/AGE] [bt/BLOOD_TYPE] [md/MEDICATION] [ht/HEIGHT] [wt/WEIGHT]
 [il/ILLNESSES] [su/SURGERIES] [fh/FAMILY_HISTORY] [ih/IMMUNIZATION_HISTORY] [gd/GENDER] [et/ETHNICITY]`
 
-* For blood type parameter, we recommend using ABO blood system, but allow any blood system or special blood type values.
+* We recommend inputting blood type using ABO blood system (eg. AB), height using centimeters (eg. 185 cm), weight using kilograms (eg. 70 kg), gender as either Male, Female or Non-binary, but allow any other system or special values
 
-Examples:
+
+* Examples:
```
Comment (joey-chance) [inline docs/UserGuide.md:282]: Should Examples be bulleted? 🤔

File: src/main/java/seedu/address/logic/parser/medical/EditMedicalCommandParser.java [hunk 2]
```diff
@@ -31,79 +30,75 @@
     /**
      * Parses the given {@code String} of arguments in the context of the EditMedicalCommand
      * and returns an EditMedicalCommand object for execution.
+     *
      * @throws ParseException if the user input does not conform the expected format
      */
     public EditMedicalCommand parse(String args) throws ParseException {
         requireNonNull(args);
-        try {
```
Comment (joey-chance) [inline src/main/java/seedu/address/logic/parser/medical/EditMedicalCommandParser.java:38]: Could you elaborate as to why the try block is not needed here?

File: src/main/java/seedu/address/model/medical/BloodType.java [hunk 3]
```diff
@@ -12,9 +12,10 @@
  */
 public class BloodType {
 
-    public static final String MESSAGE_CONSTRAINTS = "We recommend using ABO blood system, "
-            + "but allow any blood system or special blood type values. "
-            + "Blood type can take any values, and it should not be blank.";
+    public static final String MESSAGE_CONSTRAINTS = "Invalid blood type input. "
+            + "We recommend inputting blood type using ABO blood system (eg. AB), "
+            + "but allow any other blood type input values (eg. other blood system). "
```
Comment (joey-chance) [inline src/main/java/seedu/address/model/medical/BloodType.java:17]:
```suggestion
            + "but we allow any other blood type input values (eg. other blood system). "
```

File: src/main/java/seedu/address/model/medical/Gender.java [hunk 4]
```diff
@@ -9,7 +9,10 @@
  */
 public class Gender {
 
-    public static final String MESSAGE_CONSTRAINTS = "Gender can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Invalid gender input. "
+            + "We recommend inputting gender as Male, Female or Non-binary, "
+            + "but allow any other gender input values. "
```
Comment (joey-chance) [inline src/main/java/seedu/address/model/medical/Gender.java:14]:
```suggestion
            + "but we allow any other gender input values. "
```

File: src/main/java/seedu/address/model/medical/Height.java [hunk 5]
```diff
@@ -10,7 +10,10 @@
  */
 public class Height {
 
-    public static final String MESSAGE_CONSTRAINTS = "Height can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Invalid height input. "
+                + "We recommend inputting height using centimeters (eg. 185 cm), "
+                + "but allow any other height input values (eg. using other units). "
```
Comment (joey-chance) [inline src/main/java/seedu/address/model/medical/Height.java:15]:
```suggestion
                + "but we allow any other height input values (eg. using other units). "
```

File: src/main/java/seedu/address/model/medical/Weight.java [hunk 6]
```diff
@@ -10,7 +10,10 @@
  */
 public class Weight {
 
-    public static final String MESSAGE_CONSTRAINTS = "Weight can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Invalid height input. "
+            + "We recommend inputting height using kilograms (eg. 70 kg), "
+            + "but allow any other unit or special weight values (eg. using other units). "
```
Comment (joey-chance) [inline src/main/java/seedu/address/model/medical/Weight.java:15]:
```suggestion
            + "but we allow any other unit or special weight values (eg. using other units). "
```

**Review by joey-chance — DISMISSED** [review]
Please check the suggestions and let me know what you think. Everything else LGTM! 👍

File: src/main/java/seedu/address/logic/parser/medical/EditMedicalCommandParser.java (hunk 2 above)
Comment (sibinhho99-nus) [reply src/main/java/seedu/address/logic/parser/medical/EditMedicalCommandParser.java:38]: > Could you elaborate as to why the try block is not needed here?

Now the try catch block wraps all exceptions as invalid command format. We can let `ParserUtil` throw the `ParseException`s which then makes `parse` throws the same `ParseException`. The error message will then be specific to each field

**Review by joey-chance — APPROVED** [review]
LGTM! 👍

**Pull request merged**